src/Migrations.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\EncryptionApiClient;
+
+class Migrations extends Common
+{
+    public function __construct(string $sapiToken, array $config)
+    {
+        parent::__construct(['X-StorageApi-Token' => $sapiToken], $config);
+    }
+
+    public function migrateConfiguration(
+        string $sourceStorageApiToken,
+        string $destinationStack,
+        string $destinationStorageApiToken,
+        string $componentId,
+        string $configId,
+        string $branchId,
+        bool $dryRun = false,
+    ): string {
+        $queryParams = [];
+        if ($dryRun) {
+            $queryParams['dry-run'] = 'true';
+        }
+
+        $url = 'migrate-configuration';
+        if (count($queryParams) > 0) {
+            $url .= '?' . http_build_query($queryParams);
+        }
+
+        $response = $this->apiPost($url, [
+            'sourceStorageApiToken' => $sourceStorageApiToken,
+            'destinationStack' => $destinationStack,
+            'destinationStorageApiToken' => $destinationStorageApiToken,
+            'componentId' => $componentId,
+            'configId' => $configId,
+            'branchId' => $branchId,
+        ]);
+
+        return $response[0];
```

Comment:
ta 0 je co ?

Reply:
Ten endpoint na `encryption-api` vrací při 200 json response ve tvaru `["message"]`.

https://github.com/keboola/encryption-api/blob/adamvyborny-PST-1555/src/Controller/MigrateConfiguration/ConfigurationMigrator.php#L173

Reply:
a to vymyslel kdo ? :)